mysql-test/t/select_outfile_fsync_debug.test

```diff
@@ -0,0 +1,63 @@
+disable_query_log;
+-- source include/test_outfile.inc
+# Server are started in "var/master-data", so "../../tmp" will be "var/tmp"
+eval set @tmpdir="../../tmp";
```

Comment:
1. `eval` is not needed.
2. `@tmpdir` is not used anywhere.
3. May be just `$MYSQL_TMP_DIR` instead of `../../tmp`?